src/api/documentsApi.ts

```diff
@@ -0,0 +1,157 @@
+import Api from "./api";
+import notificationLogic from '../components/Notifications/Notification';
+import { successfulCreateAction, successfulEditAction, successfulDeleteAction } from "../components/Notifications/Messages"
+
+export type Document = {
+    id: number;
+    name: string;
+    type: string;
+    organization: string;
+    description: string;
+    date: string;
+    fileName: string | null;
+}
```

Comment:
It's better to extract this type to separate Model file

Reply:
Done